### PR TITLE
Feature/filter addresses

### PR DESCRIPTION
**Pull request by Jedi2002**

### CODE REVIEW

File: x/dispensation/utils/parser_test.go [hunk 1]
```diff
@@ -59,20 +61,84 @@ func init() {
...
+		panic("Err Reading file :" + err.Error())
+	}
+	err = json.Unmarshal(o, &addresStrings)
+	if err != nil {
+		panic("Err Unmarshall :" + err.Error())
+	}
+	for _, add := range addresStrings {
+		_, err := sdk.AccAddressFromBech32(add)
+		if err != nil {
+			fmt.Println("Invalid :", add)
```
Comment (iverc) [inline x/dispensation/utils/parser_test.go:101]: Should we panic here as well?

**Review by iverc — APPROVED** [review]
1 comment, otherwise lgtm

File: x/dispensation/utils/parser_test.go (hunk 1 above)
Comment (Jedi2002) [reply x/dispensation/utils/parser_test.go:101]: this test is just to print out the invalid address . So that they can be removed from the input list .

Comment (Jedi2002) [reply x/dispensation/utils/parser_test.go:101]: Its actaully a utility and not a test , jsut kept it under tests temporarily

**Comment by Jedi2002** [comment]
Closing and adding to .42

**Pull request closed without merging**